lib/google_drive/spreadsheet.rb

```diff
@@ -134,7 +134,24 @@ def batch_update(requests)
 
     # Append values to a spreadsheet by first searching for a data table at a range,
     # then appending the specified values at the end of this data table.
-    def append(range_name, values, override_params = {})
+    #
+    # +range+ The A1 notation of a range to search for a logical table of data. 
```

Comment:
I still don't understand how it determines the worksheet (i.e., tab) which the value is added to. Assuming the spreadsheet has two worksheets "Sheet 1" and "Sheet 2", and you specify "A1" here, does it add values to A1 of Sheet 1 or Sheet 2? Do you know?

Reply:
I am sorry, I don't have more information than what Google provides at https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values/append

Reply:
> I still don't understand how it determines the worksheet (i.e., tab) which the value is added to. Assuming the spreadsheet has two worksheets "Sheet 1" and "Sheet 2", and you specify "A1" here, does it add values to A1 of Sheet 1 or Sheet 2? Do you know?

You can specify "Sheet1!A1". You can find examples in the following URL.
https://developers.google.com/sheets/api/samples/writing#write_a_single_range